docs/en/02_Developer_Guides/01_Templates/07_Caching.md

```diff
@@ -35,13 +35,11 @@ When we render `$Counter` to the template we would expect the value to increase
 
 ## Partial caching
 
-Partial caching is a feature that allows the caching of just a portion of a page. Instead of fetching the required data
-from the database to display, the contents of the area are fetched from a [cache backend](../performance/caching).
+Partial caching is a feature that allows caching of a portion of a page as a single string value. For more details read [its own documentation](partial_template_caching).
 
+Example:
 ```ss
-<% cached 'MyCachedContent', LastEdited %>
-    $Title
+<% cached 'MyCachedContent', $LastEdited if $IsCacheable %>
```

Comment:
I don't think that `if $IsCacheable` is a good example here:
- There's no generic function on DataObject `IsCacheable()` so it's confusing (I had a to double check this myself)
- Why would you even do a check here?  Templates are rigid and you know ahead of time what type of DataObject you're dealing with, so if you don't want to cache it, you simply wouldn't add the `cached` block to begin with

Reply:
> There's no generic function on DataObject IsCacheable()

We're not claiming that it's getting rendered on the generic "DataObject". The variable name is chosen to demonstrate the purpose of that element within the context.

> Why would you even do a check here?

This is demonstration of the `cached` block syntax.

Reply:
Have an example in official docs implies it does exist.  Most people will skim read docs and basically just copy paste examples into their own code to get things working with minimum effort on their part.  If they copy paste this example it won't work.

> We're not claiming that it's getting rendered on the generic "DataObject"

The example has `$LastEdited` which would be called against a DataObject.  So it's pretty confusing

Reply:
We are talking out of context of a particular object implementation. We need a way to document those syntactic elements. `$IsCacheable` in my opinion is well self-descriptive and easy to understand.